pkg/service/v0/response/response.go

```diff
@@ -38,15 +37,15 @@ func (p Payload) MarshalXML(e *xml.Encoder, start xml.StartElement) (err error)
 	}
 
 	// encode the meta tag
-	if err = e.EncodeElement(p.Meta, metaStartElement); err != nil {
+	if err = e.EncodeElement(rsp.OCS.Meta, metaStartElement); err != nil {
 		return
 	}
 
 	// we need to use reflection to determine if p.Data is an array or a slice
```

Comment:
Why does this matter?

Reply:
I'm not sure what will happen if one would not do it. If the data is an array or slice we iterate to encode each item in the array/slice into its own xml element. I would assume that this would happen automatically but maybe not. @butonic surely had his reasons to do it like that.